src/components/Quiz/index.tsx

```diff
@@ -1,26 +1,27 @@
-import * as classes from "./style.module.css";
+import * as classes from './style.module.css';
 
-export function Quiz() {
-  const handleClick = () => {
-    console.log("Clicked");
-  };
+import { Answer } from '~/src/components/Answer';
 
+type Props = {
+  possibleAnswers: {
+    id: number;
+    label: string;
+    isCorrect: boolean;
+  }[];
```

Comment:
Extraia o tipo do objeto de opção pra um tipo próprio, `Option`.

To mudando de "answer" pra "option" onde eu vejo porque "answer" tem mais o sentido de "a opção escolhida como resposta" e "option" tem mais o sentido de "opções disponíveis para escolha".